README.md

```diff
@@ -72,3 +72,6 @@ export default class FlashMessages {
 - removeClass
 - visit
 - submit
+- url
+- urlParams
+- getParam
```

Comment:
@AlexLarra could you please add here method signature? I think it helps a lot to understand what's going on under the hood